doc/architecture/vulnerability-management.rst

```diff
@@ -0,0 +1,146 @@
+ist.gov/cvss.cfm?calculator&adv&version=2>`_. Based on the result of the
+CVSS tool and the Ostro Project security team’s own vulnerability
+assessment, the security team takes action for getting the vulnerability
+fixed.
+
+If the vulnerability is considered to be critical, a typical response
+is adding a patch to ``meta-ostro-fixes`` repository, releasing the
+patch in ``ostro-os`` repository, reporting the bug to the meta-layer
+which provides the component in question, and sending out an Ostro OS
+security advisory. The patch either updates the component in question
+to a version which doesn’t have the vulnerability or backports the fix
+if moving to a new component version is not desired. If the component
+cannot be fixed quickly, the security team needs to consider disabling
+it from Ostro OS temporarily. If the security vulnerability is critical
+enough and it is estimated to put existing Ostro OS -based devices in
+serious danger, Ostro OS can make an unscheduled release to get the bug
+fix quickly to a wider audience.
+
+If the vulnerability is not critical and it’s expected that upstream
+meta-layer providing the component will handle it promptly, it is often
+enough to make sure the upstream meta-layer is aware of the issue, and
+then just wait for a fix to be delivered via the normal update mechanism
+into Ostro OS. It's understood that the default mechanism for how
+security fixes come to Ostro OS is via the upstream meta-layers.
+
+If the vulnerability is in Ostro OS core, the problem needs to be fixed
+in Ostro OS itself, and the upstream considerations are not necessary.
+
+Finally, the processing of the vulnerability leads to a (periodic) look
+at Ostro OS security architecture. The Ostro Project security team has
+to consider if Ostro OS security architecture needs be adapted to
+better protect against the vulnerability and others like it. This is
+continuous work and eventually lead to an Ostro OS security
+architecture that is well prepared to handle many vulnerability
+classes.
+
+Tracking and discussing the security vulnerability and the fix to it
+will happen in the Ostro OS issue tracker. The security vulnerabilities
+in the issue tracker need to be classified so that it’s easy to see the
+distribution’s security vulnerability status by doing a single search in
+the issue tracker.
+
+Getting updates to client devices
+=================================
+
+The Ostro Project doesn't provide binaries to end users, meaning people
+who run devices which have Ostro OS -based distributions. Instead the
+Ostro OS customers are ODMs, which create and distribute binaries to the
+devices. For this reason, it’s paramount that the ODMs actively track
+Ostro OS security issues and also deliver security updates to their end
+users via the normal software update process.  Ostro OS is an
+open-source project, so Ostro Project itself does not necessarily know
+which ODMs actually use Ostro OS code in their devices.
```

Comment:
So, you need to reflect two points in the doc: 1) images that are published are not for products and should not be used for products (this part is present now) and 2) however, ostro is nice enough to publish updates for these images if one wants to pull them manually for some reason. How often will updates be published? right when any component is changed in master or only in case we have significant vulnerability? I think this needs to be reflected also.